index.html

```diff
@@ -160,10 +148,25 @@ <h1>The OpenTF Manifesto</h1>
         </ul>
 
         <p class="font-bold">
-          We ask HashiCorp to do right by the community and donate Terraform to
-          a reputable foundation where, together with the community and
-          resources that we pledge, we continue to build and maintain this
-          amazing ecosystem.
+          Our request of HashiCorp: donate Terraform to this foundation.
+        </p> 
+        
+        <p>
+          We ask HashiCorp to do the right thing by the community and to donate Terraform to this foundation,
+          rather than going forward with the BSL license change. That way, instead of fracturing the community, 
+          we end up with a single, impartial, reliable home for Terraform where the whole community can come together 
+          to keep building this amazing ecosystem.
+        </p>        
+
+        <p class="font-bold">
+          Our fallback plan: fork Terraform.
```

Comment:
I don't think we should mention this part.  We should just be asking to donate.

Reply:
Agreed.

Reply:
Then what are we pledging to do at the end?

To contribute to the foundation if HashiCorp donates to it? 

And if they choose not to? Or just ignore the manifesto?

Don't manifesto's typically promise action?

Reply:
To contribute to the foundation regardless of whether Hashi donates it or not.

Reply:
IMO we write this with the assumption that HCP will donate it.